karspexet/ticket/models.py

```diff
@@ -46,10 +52,36 @@ class Ticket(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     last_modified_at = models.DateTimeField(auto_now=True)
 
+    def __repr__(self):
+        return "<Ticket %s | %s | %s>" % (self.ticket_type, self.show, self.seat)
```

Comment:
Since this is in Python 3, you should use formatting strings, or string.format:

```python
f"<Ticket {self.ticket_type!r} | {self.show!r} | {self.seat!r}>"
```

The `!r` is to use `repr` instead of `str`. By putting a `f` infront of a string it basically turns into Ruby's `#{}` or JavaScripts template strings \`${}\`. (And yes, those are backticks.)